style.css

```diff
@@ -446,3 +497,312 @@ input[type="submit"] {
   background: #0d151e;
   border-radius: 10px;
 }
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] At exactly `min-width: 768px`, your app displayed properly but I noticed there is an horizontal overflow which doesn't give a good user experience. Your app will look more professional if you can work on this. Thanks. 👍